app/screens/user_profile/user_info.tsx

```diff
@@ -17,36 +20,49 @@ type Props = {
     showNickname: boolean;
     showPosition: boolean;
     user: UserModel;
+    enableCustomAttributes?: boolean;
 }
 
-const UserInfo = ({localTime, showCustomStatus, showLocalTime, showNickname, showPosition, user}: Props) => {
-    const {formatMessage} = useIntl();
+const UserInfo = ({localTime, showCustomStatus, showLocalTime, showNickname, showPosition, user, enableCustomAttributes}: Props) => {
     const customStatus = getUserCustomStatus(user);
+    const serverUrl = useServerUrl();
+    const [customAttributes, setCustomAttributes] = useState<DisplayCustomAttribute[]>([]);
+
+    useEffect(() => {
+        if (enableCustomAttributes) {
+            const fetchData = async () => {
+                try {
+                    const client = NetworkManager.getClient(serverUrl);
+                    const fields = await client.getCustomProfileAttributeFields();
+                    const attrValues = await client.getCustomProfileAttributeValues(user.id);
+                    logDebug('got fields: ', fields.length);
+                    logDebug('got values: ', attrValues.length);
+                    if (fields) {
+                        const attributes = fields.map((field) => ({
+                            id: field.id,
+                            name: field.name,
+                            value: attrValues[field.id] || '',
+                        }));
+                        setCustomAttributes(attributes);
+                    }
+                } catch (error) {
+                    setCustomAttributes([]);
+                }
+            };
+
+            fetchData();
+        }
+    }, [enableCustomAttributes, serverUrl, user.id]);
 
     return (
         <>
-            {showCustomStatus && <UserProfileCustomStatus customStatus={customStatus!}/> }
-            {showNickname &&
-                <UserProfileLabel
-                    description={user.nickname}
-                    testID='user_profile.nickname'
-                    title={formatMessage({id: 'channel_info.nickname', defaultMessage: 'Nickname'})}
-                />
-            }
-            {showPosition &&
-                <UserProfileLabel
-                    description={user.position}
-                    testID='user_profile.position'
-                    title={formatMessage({id: 'channel_info.position', defaultMessage: 'Position'})}
-                />
-            }
-            {showLocalTime &&
-                <UserProfileLabel
-                    description={localTime!}
-                    testID='user_profile.local_time'
-                    title={formatMessage({id: 'channel_info.local_time', defaultMessage: 'Local Time'})}
-                />
-            }
+            {showCustomStatus && <UserProfileCustomStatus customStatus={customStatus!}/>}
+            <CustomAttributes
+                nickname={showNickname ? user.nickname : undefined}
```

Comment:
You can remove the ternary:
```suggestion
                nickname={showNickname && user.nickname}
```

Same goes with the rest

Reply:
Actually, this causes a type error, so the ternary is needed. showNickname is a `boolean` so the `&&` operator would resolve to `false` instead of `string?`

Reply:
Ah, thanks!